pulp_ansible/app/tasks/utils.py

```diff
@@ -9,12 +9,18 @@
 from pulp_ansible.app.constants import PAGE_SIZE
 
 
-def get_page_url(url, page=1):
+def get_page_url(url, page=1, api_version=None):
```

Comment:
`api_version` I think should be a required param.

Reply:
`get_page_url` is used by roles task, I made it optional because I wasn't sure if I have to change roles task

Reply:
https://github.com/pulp/pulp_ansible/blob/master/pulp_ansible/app/tasks/roles.py#L133-L145